helm/alb-ingress-controller/README.md

```diff
@@ -0,0 +1,79 @@
+# alb-ingress-controller
+
+[alb-ingress-controller](https://github.com/coreos/alb-ingress-controller) satisfies Kubernetes ingress resources by provisioning Application Load Balancers and Route53 Resource Record Sets.
+
+## TL;DR:
+
+```console
+$ helm install stable/alb-ingress-controller
+```
+
+## Introduction
+
+This chart bootstraps an alb-ingress-controller deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/alb-ingress-controller
+```
+
+The command deploys alb-ingress-controller on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the alb-ingress-controller chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`aws.accessKeyId` | If provided, AWS_ACCESS_KEY_ID environment variable will be set to this value | `""`
+`aws.accessSecretKey` | If provided, AWS_ACCESS_SECRET_KEY environment variable will be set to this value | `""`
+`aws.debug` | If true, enables logging on all outbound AWS API requests | `false`
+`aws.region` | (REQUIRED) AWS region in which this ingress controller will operate | `us-east-1`
```

Comment:
Any chance we can use `us-west-1` just to make it consistent with `alb-ingress-example` at https://github.com/coreos/alb-ingress-controller/blob/master/examples/alb-ingress-controller.yaml#L50-L51.

If not, I can switch the example to us-east-1